src/scss/panels/MenuPanel.scss

```diff
@@ -23,6 +23,37 @@
     }
   }
 
+  .report {
+    a, button {
+      font-size: 14px;
+      box-shadow: none;
+      transition: none;
+      border: none;
+      padding: 0 5px;
+      margin-bottom: 4px;
+
+      &:hover {
+        cursor: default;
```

Comment:
We need to talk about this some time :-) It's not very user friendly to prevent the pointer cursor to be displayed on a clickable element, but I am aware that it's consistent with the rest of the app.